eng/Signing.props

```diff
@@ -19,13 +19,29 @@
     
     <!-- Binary test asset -->
     <FileSignInfo Include="testwpf.dll" CertificateName="None" />
+
+    <!-- Files in the layout that should not be signed -->
+    <FileSignInfo Include="apphost.exe" CertificateName="None" />
```

Comment:
Will this need to be added to the signcheck exclusions file?

Reply:
It should be in there already (somewhere). It's never signed today.